core/trino-main/src/test/java/io/trino/connector/MockConnectorInsertTableHandle.java

```diff
@@ -13,9 +13,48 @@
  */
 package io.trino.connector;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import io.trino.spi.connector.ConnectorInsertTableHandle;
+import io.trino.spi.connector.SchemaTableName;
 
-class MockConnectorInsertTableHandle
+import java.util.Objects;
+
+import static java.util.Objects.requireNonNull;
+
+public class MockConnectorInsertTableHandle
         implements ConnectorInsertTableHandle
 {
+    private final SchemaTableName tableName;
+
+    @JsonCreator
+    public MockConnectorInsertTableHandle(@JsonProperty SchemaTableName tableName)
```

Comment:
Since only one argument is present. I guess Jackson should easily map the it so that roundtrip happens without any issues. 
Will add the argument to the annotation